docs/examples/scheduled-capacity.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: autoscaling.karpenter.sh/v1alpha1
```

Comment:
As an alternative idea that I think would be easier to parse and easier to understand, I sketched out the following (it's just a sketch) - happy to discuss offline:

```yaml
kind: MetricsProducer
metadata:
  name: scheduling
spec:
  scheduledCapacity:
    defaultReplicas: 2
    timezone: America/Los_Angeles
    schedules:
      # Highest priority overrides everything else. I do this because I happen
      # to know a bunch of employees of MyCompany are going to be using this cluster
      # during this upcoming time.
      - priority: 1000
        start: year:2020 month:June day:23 hour:10 minute:23
        end: year:2020 month:June day:23 hour:19 minute:00
        replicas: 15
      # Scale way down on Friday evening for the weekend
      - start: dayOfWeek:Friday hour:19
        replicas: 1  
      # Scale up on Weekdays for usual traffic
      - start: dayOfWeek:MTWHF hour:6
        replicas: 7
      # Scale down on weekday evenings but not as much as on weekends
      # cuz a few ppl are still using systems
      - start: dayOfWeek:MTWHF hour:17
        replicas: 3
```